DuckDuckGo/SettingsViewController.swift

```diff
@@ -50,11 +50,14 @@ class SettingsViewController: UITableViewController {
     @IBOutlet weak var textSizeCell: UITableViewCell!
     @IBOutlet weak var textSizeAccessoryText: UILabel!
     @IBOutlet weak var widgetEducationCell: UITableViewCell!
+    @IBOutlet weak var autofillCell: UITableViewCell!
     
     @IBOutlet var labels: [UILabel]!
     @IBOutlet var accessoryLabels: [UILabel]!
     
-    private let defaultBroswerSectionIndex = 0
+    private let defaultBrowserSectionIndex = 0
+    private let autofillSectionIndex = 1
```

Comment:
Increasingly feel like we should have a better system for this (e.g. an enum) but given the extent we want to yeet this entire class I think it's fine